### DISCUSSION TITLE
Static Overlay

### DISCUSSION THREAD

**Discussion by morganite-proto** [noisedeck/noisedeck-docs#24] [Ideas]
The ability to place an overlay image on the output would be helpful for VJs. Right now I'm making some visuals to be set with music, and I'm adding a cutout of a logo in Premiere Pro. Here's a quick example of what I'm doing with the preset "underwater kaleidoscope": 

https://user-images.githubusercontent.com/7952127/120123372-2b320e80-c174-11eb-8239-c0b71bc7be3b.mp4

**Comment by jesshewitt** [comment]
We can do this! I'll add it to the feature requests, and we'll try to get it into a release in the near future.
